src/js/pages/Index.js

```diff
@@ -40,6 +41,7 @@ var Index = React.createClass({
   componentWillMount() {
     MetadataStore.init();
     SidebarStore.init();
+    MesosStateStore.addChangeListener(EventTypes.MESOS_STATE_CHANGE, () => {});
```

Comment:
I know we never unsubscribe, but shouldn't we at least remove the listener when the component unmounts? My reason being if for any reason someone/something starts mounting unmounting the component, I think we can run on a memory leak where those callbacks only get added to the list of listeners (assuming here it is a list of it).

Reply:
👉 poke